tests/Support/NovaTest/NovaTestMockTrait.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+namespace RonasIT\Support\Tests\Support\NovaTest;
+
+use org\bovigo\vfs\vfsStream;
+use RonasIT\Support\Generators\NovaTestGenerator;
+use RonasIT\Support\Tests\Support\Shared\GeneratorMockTrait;
+use RonasIT\Support\Traits\MockClassTrait;
+
+trait NovaTestMockTrait
+{
+    use GeneratorMockTrait, MockClassTrait;
+
+    public function mockNovaResourceTestGenerator(): void
+    {
+        $this->mockClass(NovaTestGenerator::class, [
+            [
+                'method' => 'getModelFields',
```

Comment:
let's mock only methods which result can not been changed from the third-party factors, e.g. mocking `file_exists` method is good.

For this case - I belive we should mock the `getFields` fethod of called model, or make a test model with the `getFields` methods

Reply:
@DenTray the logic for getting model fields is complicated and will be covered in different test cases in a future request. For now, I include in the test minimum logic to assert only generator results.